pkg/tls/tls.go

```diff
@@ -52,7 +52,7 @@ var (
 func ListenAndServe(ctx context.Context, restConfig *rest.Config, handler http.Handler, bindHost string, httpsPort, httpPort int, acmeDomains []string, noCACerts bool) error {
 	restConfig = rest.CopyConfig(restConfig)
 	restConfig.Timeout = 10 * time.Minute
-	opts := &server.ListenOpts{}
+	opts := &server.ListenOpts{DisplayServerLogs: true}
```

Comment:
I don't think this fully works. I see this getting overridden on L68, and the internal listener uses a different set of options that seems to be constructed separately (on L105). Can you give this another look to make sure that we cover these cases too?

Reply:
yes, good catch.